src/components/Header/Header.scss

```diff
@@ -1,9 +1,142 @@
 @import '~style/variables';
 
-.wrapper {
-  border: 1px solid blue;
+header {
+  display: flex;
+  flex-direction: column;
+  background-color: $color-primary--500;
+  color: $color-white;
+
+  a {
+    color: $color-white;
+    text-decoration: underline;
+    text-decoration-color: $color-highlight;
+  }
+}
+
+.govContainer {
+  display: flex;
+  height: 48px;
+  padding-left: 14px;
+  flex-flow: column wrap;
+  align-items: flex-start;
+  align-content: flex-start;
+  justify-content: center;
+
+  @include for-desktop-up {
+    padding-left: 30px;
+    height: 32px;
+    align-items: center;
+    flex-flow: row nowrap;
+    justify-content: flex-start;
+    background-color: $color-primary--400;
+
+    > * {
+      margin-right: 8px;
+    }
+  }
+}
+
+.compactTitle,
+.compactSubtitle {
+  @include for-desktop-up {
+    display: none;
+  }
+}
+
+.govAu,
+.govWebsite {
+  display: none;
+
+  @include for-desktop-up {
+    display: initial;
```

Comment:
This is trickier. Technically `display: inline;` will work, but will break if those spans become anything but spans. Good enough for now I'd say.